spec/02-integration/05-proxy/01-proxy_spec.lua

```diff
@@ -43,7 +43,8 @@ describe("Proxy interface listeners", function()
       proxy_listen = "off",
       admin_listen = "0.0.0.0:9001",
     }))
-    assert.equals(1, count_server_blocks(helpers.test_conf.nginx_kong_conf))
+    assert.equals(2, count_server_blocks(helpers.test_conf.nginx_kong_conf))
+    --assert.equals(1, count_server_blocks(helpers.test_conf.nginx_kong_conf))
```

Comment:
Leaving the old lines as comments doesn't make it more clear to me, I suggest removing.

Reply:
done